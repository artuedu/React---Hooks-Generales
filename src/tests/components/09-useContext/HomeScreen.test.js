import '@testing-library/jest-dom';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { HomeScreen } from '../../../Components/09-useContext/HomeScreen';
import { UserContext } from '../../../Components/09-useContext/UserContext';

describe('ruebas en <HomeScreen />', () => {

    const user = {
        name: 'Arturo',
        email: 'arturo@correo.com'
    };
    
    const wrapper = mount(
        <UserContext.Provider value={{
            user
        }}>
            <HomeScreen />
        </UserContext.Provider>
    );
    
    test('debe de mostrarse correctamente', () => {
        
        expect(wrapper).toMatchSnapshot();

    });
    
});
