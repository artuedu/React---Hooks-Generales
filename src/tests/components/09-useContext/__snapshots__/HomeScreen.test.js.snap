// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ruebas en <HomeScreen /> debe de mostrarse correctamente 1`] = `
<div>
  <h1>
    HomeScreen
  </h1>
  <hr />
  <pre>
    {
   "name": "Arturo",
   "email": "arturo@correo.com"
}
  </pre>
</div>
`;
